docs/products/tools/marketplace/guides/hashicorp-nomad-clients-cluster/index.md

```diff
@@ -0,0 +1,60 @@
+---
+description: "Deploy HashiCorp Nomad Clients Cluster, a flexible scheduling and orchestration for diverse workloads, on Linode Compute Instances.'"
+keywords: ['HashiCorp','Nomad','Scheduling', 'orchestration']
+tags: ["marketplace", "linode platform", "cloud manager"]
+published: 2023-09-11
+modified_by:
+  name: Linode
+title: "Deploy HashiCorp Nomad Clients Cluster through the Linode Marketplace"
+authors: ["Linode"]
+---
+
+[HashiCorp Nomad](https://www.nomadproject.io/) is a workload scheduler and orchestrator that allows you to deploy and manage containerized or non-containerized, legacy applications using a single, unified workflow. Nomad uses bin packaging for efficient job scheduling and resource optimization, as well as enables developers to use declarative infrastructure-as-code for their deployments.
```

Comment:
The opening paragraph for this and the other OCC should address their differences.

Reply:
Edited!